### PR TITLE
BROKERS: List Files

### LINKED ISSUE #354: BROKERS: List Files

### PR DESCRIPTION
**Pull request by diogod3**
Closes #354

### CODE REVIEW

File: Standard.AI.OpenAI/Brokers/OpenAIs/IOpenAIBroker.AIFiles.cs [hunk 1]
```diff
@@ -13,5 +13,7 @@ internal partial interface IOpenAIBroker
             ExternalAIFileRequest externalFileRequest);
 
         ValueTask<ExternalAIFileResponse> DeleteFileByIdAsync(string fileId);
+
```
Comment (hassanhabib) [inline Standard.AI.OpenAI/Brokers/OpenAIs/IOpenAIBroker.AIFiles.cs:16]: No empty lines between one-liners

**Comment by hassanhabib** [comment]
@diogod3 PR name must match issue

**Comment by diogod3** [comment]
Sorry, forgot to fix the PR name

**Comment by diogod3** [comment]
Request made using broker:

image

File: Standard.AI.OpenAI/Models/Services/Foundations/ExternalAiFiles/ExternalAIFilesResult.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+
+using Newtonsoft.Json;
+
+namespace Standard.AI.OpenAI.Models.Services.Foundations.ExternalAiFiles
```
Comment (hassanhabib) [inline Standard.AI.OpenAI/Models/Services/Foundations/ExternalAiFiles/ExternalAIFilesResult.cs:7]: Rename the name space to `ExternalAIFiles` not `Ai`

**Pull request merged**